code/game/objects/items/plushes.dm

```diff
@@ -474,10 +473,6 @@
 	if(P && istype(P.loc, /turf/open) && !P.clash_target && !clashing)
 		P.clash_of_the_plushies(src)
 
-/obj/item/toy/plush/narplush/hugbox
-	desc = "A small stuffed doll of the elder goddess Nar'Sie. Who thought this was a good children's toy? <b>It looks sad.</b>"
-	is_invoker = FALSE
-
```

Comment:
I feel like this probably is causing some bugs TG-side, with this whole type being just missing.